cloudvolume/meshservicegraphene.py

```diff
@@ -146,7 +169,11 @@ def get(self, seg_id, remove_duplicate_vertices=False):
         meshdata = []
         for frag in tqdm(fragments, disable=(not self.vol.progress),
                          desc="Decoding Mesh Buffer"):
-            mesh = decode_mesh_buffer(frag["content"])
+            try:
+                # EAFP
```

Comment:
What does EAFP mean?

Reply:
Thinking about this a bit more, I think it would be cleaner to provide a `DracoPy.is_draco_file` method that returns a boolean, or throw a custom exception like DracoPy.FileTypeException as a ValueError could be generated internal to Draco and provides less certainty that taking the standard path is actually correct.

Reply:
Agreed. I will fix this in the next commit